### PR TITLE
Migrate reply and reply all icon to material design

### PR DESCRIPTION
**Pull request by GretaD**
ref https://github.com/nextcloud/groupware/issues/38

before
buttonold

after
afterbutton

### CODE REVIEW

**Comment by ChristophWurst** [comment]
FYI https://github.com/nextcloud/server/pull/25774 requires https://github.com/nextcloud/server/pull/25774. This means we have a hard dependency on Nextcloud 23 with buttons from nextcloud-vue. If I'm not mistaken they will not render correctly with 22. Give it a try and if it works then proceed, otherwise we need to wait for the drop of Nextcloud 22.

**Comment by GretaD** [comment]
> Give it a try and if it works then proceed, otherwise we need to wait for the drop of Nextcloud 22.

You mean to give it a try with Vue `<Button>` ?

**Comment by ChristophWurst** [comment]
Is the button component a hard requirement for icons withing buttons?

**Comment by GretaD** [comment]
> Is the button component a hard requirement for icons within buttons?

to answer your question, no its not a hard requirement, but in this case im stupid, i imported the Button with the wrong path and assumed its not released yet.

File: src/components/ThreadEnvelope.vue [hunk 1]
```diff
@@ -71,16 +71,18 @@
...
 			<div class="right">
 				<Moment class="timestamp" :timestamp="envelope.dateInt" />
-				<button
-					:class="{
-						'icon-reply-all-white': hasMultipleRecipients,
-						'icon-reply-white': !hasMultipleRecipients,
-						primary: expanded,
-					}"
-					class="button"
+				<Button
```
Comment (ChristophWurst) [inline src/components/ThreadEnvelope.vue:74]: did you make a screenshot of your Nextcloud 22 test by any chance? I wonder how the button renders

Comment (GretaD) [reply src/components/ThreadEnvelope.vue:74]: yes, check the description

**Comment by ChristophWurst** [comment]
Please squash

**Pull request merged**